knowhere/common/Utils.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2019-2020 Zilliz. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under the License
+
+#include "common/Log.h"
+#include "common/Utils.h"
+
+namespace knowhere::utils {
+
+namespace {
+const int64_t kDefaultBuildOmpNum = omp_get_max_threads();
+const int64_t kDefaultQueryOmpNum = omp_get_max_threads();
+}  // namespace
+
+void
+SetBuildOmpThread(const Config& conf) {
```

Comment:
write this API in this way, it's better:
```
void
SetBuildOmpThread(const Config& conf) {
    int32_t omp_num = CheckKeyInConfig(conf, meta::BUILD_INDEX_OMP_NUM) ? 
                                    GetMetaBuildIndexOmpNum(conf) :
                                    kDefaultBuildOmpNum;
    omp_set_num_threads(kDefaultBuildOmpNum);
    LOG_KNOWHERE_DEBUG_ << "Set current omp thread num for build: " << omp_num;
}
```

Reply:
ok